app/views/slices/image.php

```diff
@@ -1,17 +1,18 @@
 <?php 
-$imageWithCaption = $slice->getPrimary();
+use Prismic\Dom\RichText;
+
+$imageWithCaption = $slice->primary;
         
-$imageUrl = $imageWithCaption->getImage('image') ? $imageWithCaption->getImage('image')->getUrl() : '';
-$caption = $imageWithCaption->get('caption') ? $imageWithCaption->getText('caption') : null;
+$imageUrl = $imageWithCaption->image->url;
+$caption = $imageWithCaption->caption;
 
-// Render the correct style for the different image labels 
-switch ( $slice->getLabel() ) {
+switch ( $slice->slice_label ) {
 
   // Full width image
   case "image-full-width":
     echo "<div class='blog-header single' style='background-image: url(" . $imageUrl . ");'><div class='wrapper'>";
     if ( $caption ) {
-      echo "<h1>" . $caption . "</h1>";
+      echo "<h1>" . RichText::asText($caption) . "</h1>";
```

Comment:
I would probably do the `RichText::asText()` up above when setting the `$caption` variable. This isn't a huge deal, though.